pkg/kubelet/container/testing/fake_runtime.go

```diff
@@ -368,8 +381,18 @@ func (f *FakeRuntime) GetImageRef(_ context.Context, image kubecontainer.ImageSp
 	defer f.Unlock()
 
 	f.CalledFunctions = append(f.CalledFunctions, "GetImageRef")
+	imageMapping := f.ImageSpecToImageMapping
+	if imageMapping == nil {
+		imageMapping = defaultImageSpecToImageMapping
+	}
+
+	if img, ok := f.ImageList[imageMapping(image).ID]; ok {
+		return img.ID, nil
+	}
+
 	for _, i := range f.ImageList {
-		if i.ID == image.Image {
+		if slices.Index(i.RepoDigests, image.Image) > -1 ||
+			slices.Index(i.RepoTags, image.Image) > -1 {
```

Comment:
RepoDigests and RepoTags contain the Image? I thought digests / tags were distinct from the image